src/orchestration/Orchestration.ts

```diff
@@ -473,6 +488,7 @@ export class Orchestration {
             },
             [TelemetryEnum.Interaction]: (config: object): InternalPlugin[] => {
                 return [new DomEventPlugin(config)];
+                return [new JsEventPlugin(config)];
```

Comment:
Should be a list of both plugins, not two return statements.

Reply:
you are correct :-)